packages/core/package.json

```diff
@@ -22,7 +22,7 @@
     "audit": "better-npm-audit audit -l high -p",
     "prepublishOnly": "npm run test && npm run build",
     "audit-fix": "npm audit fix",
-    "test": "jest"
+    "test": "jest --verbose"
```

Comment:
I don't think verbose should be here all the time, it's something we want to use only for debug, not when all the tests passes.

Reply:
I was trying to get more info in the out put from the ci to make sure the contract test had run with the right env vars